### DISCUSSION TITLE
File doesn't compile correctly. I get - {"sources": {"SimpleStorage.sol": {"id": 0}}}

### DISCUSSION THREAD

**Discussion by SiKi02** [smartcontractkit/full-blockchain-solidity-course-py#1033] [Q&A] [answered]
```Python
from solcx import compile_standard
#install_solc
#install_solc("0.6.0")
import json

with open("./SimpleStorage.sol", "r") as file:
    simple_storage_file = file.read()
    
# compile solidity

compiled_sol = compile_standard(
    {
    "language": "Solidity",
    "sources": {"SimpleStorage.sol": {"content": simple_storage_file}},
    "settings": {
        "outputSelection": {
            "*": {
                "+": ["abi", "metadeta", "evm-bytecode", "evm.sourceMap"]
            }
        }
    }
}, 
solc_version = "0.6.0"
)
```

```
with open("compiled_code.json", "w") as file:
    json.dump(compiled_sol, file
```
<img width="1440" alt="Captura de Pantalla 2022-02-11 a la(s) 7 02 33 p  m" src="https://user-images.githubusercontent.com/96997244/153689814-983864fb-5f17-4b2e-aa3a-e5cd7c13b2c7.png">
<img width="1440" alt="Captura de Pantalla 2022-02-11 a la(s) 7 02 35 p  m" src="https://user-images.githubusercontent.com/96997244/153689819-f400e0a5-fa9c-4c3b-942b-579a945cb385.png">

**Comment by cromewar** [comment, marked as answer]
hello @SiKi02 
You have a typo, you are using "metadeta" and the correct word is "metadata".
Give it  a try ;)

**Reply by SiKi02** [reply]
silly me bro, thank you very much!
